### ISSUE TITLE
Add some medical image conferences

### ISSUE THREAD

**Issue by fepegar** [yuchenlin/rebiber#6]
I can work on this soon. The main one is [MICCAI](https://dblp.org/db/conf/miccai/index.html).

**Comment by fepegar** [comment]
The extensions in the README are not coherent with the ones in `data/`. Is this on purpose? Just to be sure when opening the PR.

**Comment by yuchenlin** [comment]
Thank you very much! 

> The extensions in the README are not coherent with the ones in `data/`. Is this on purpose? Just to be sure when opening the PR.

Both `data/*.json` and `data/*.bib.json` are acceptable as long as the correct path is in the `bib_list.txt`.

**Comment by fepegar** [comment]
Might be a silly question, but I'm not sure how to perform "Step 2: Download the bib file". Where do I need to click?

**Comment by yuchenlin** [comment]
Hi @fepegar , I will make the readme more clear. There is a button like this:

![image](https://user-images.githubusercontent.com/10104354/106265307-af32b480-61db-11eb-8833-1c5ea2c34176.png)

**Comment by fepegar** [comment]
Thanks! I tried that, but, as the screenshot says, it exports the first 1000 hits only.

**Comment by yuchenlin** [comment]
Yes, this is painful. I was wondering there is any DBLP API that can help us. I am now busy with an urgent deadline. And if you would like to explore a little bit about that, it would be very helpful to all users!

**Comment by fepegar** [comment]
I also have a deadline on Monday. I'll let you know if I find out anything.

**Comment by yuchenlin** [comment]
> I also have a deadline on Monday. I'll let you know if I find out anything.

Sure thanks! Good luck!

**Comment by fepegar** [comment]
For you too!

**Comment by yuchenlin** [comment]
Partial progress: #70 adds `miccai` to the default DBLP download list (`python rebiber/download_dblp.py --confs miccai`).

**Not done:** no MICCAI (or other medical-imaging) JSON is in #69 yet, so rebiber still will not rewrite those cites until someone runs the downloader and commits the dumps. Leaving open.

**Comment by fepegar** [comment]
I completely forgot about all of this. Apologies!

Please let me know if I can help.

**Comment by yuchenlin** [comment]
Status update from current `main`:

**Done:** MICCAI **2022–2025** main-track dumps are in the packaged data (`data/miccai2022.bib.json` … `miccai2025.bib.json`, listed in `bib_list.txt`). The DBLP downloader treats `miccai` as a multi-volume LNCS toc. Workshop tocs are not included.

**Still open:** nothing before 2022, and **ISBI** is still missing (not in `bib_list.txt` / `data/`).

@fepegar do you still want pre-2022 MICCAI and/or ISBI dumps, or is the 2022–2025 main track enough for this issue?

**Comment by fepegar** [comment]
Am I speaking with a bot? If yes, keep getting stuff.
